src/Accordion/MoreLessButton/MoreLessButton.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styles from './MoreLessButton.scss';
+import Button from '../../Button';
+import ChevronDown from '../../new-icons/ChevronDown';
+import ChevronUp from '../../new-icons/ChevronUp';
+import classNames from 'classnames';
+import TextButton from '../../TextButton';
+import { BUTTON_TYPES } from '../constants';
+
+export default ({
+  dataHook,
+  isOpen,
+  handleClick,
+  expandLabel,
+  collapseLabel,
+  hover,
+  buttonType,
+}) => {
+  const toggleStyle = classNames(styles.toggleButtons, {
+    [styles.open]: isOpen,
+  });
+
+  const buttonStyle = classNames(styles.button, {
+    [styles.active]: isOpen,
+  });
+
+  const shouldRenderButton =
+    hover && !isOpen && buttonType === BUTTON_TYPES.BUTTON;
+  const shouldRenderTextButton =
+    hover && !isOpen && buttonType === BUTTON_TYPES.TEXT_BUTTON;
+  return (
+    <div
+      data-hook={dataHook}
+      className={toggleStyle}
+      onClick={e => {
```

Comment:
`handleClick` prop should be named `onClick`, then

```suggestion
      onClick={onClick}
```

the `stopPropagation`, if really needed, should be handled by consumer of `MoreLessButton` component

Reply:
If we remove the `stopPropagation` then the Less button won't work. Therefore I think it should be inside the `MoreLessButton` and not handled by the consumer.

Reply:
ok, ignore it for now